balancer/weightedroundrobin/balancer_test.go

```diff
@@ -594,16 +619,16 @@ func (s) TestBalancer_AddressesChanging(t *testing.T) {
 
 	// srv1: weight 10
 	srv1.oobMetrics.SetQPS(10.0)
-	srv1.oobMetrics.SetCPUUtilization(1.0)
+	srv1.oobMetrics.SetApplicationUtilization(1.0)
```

Comment:
Optional since you already test this in TestBalancer_TwoAddresses_ReportingEnabledOOB. Can you leave some of these SetCPUUtilization calls to make sure it still works, and also test the precedence ordering if both are set as well. Actually I'm fine with leaving all these tests to that test case, it tests all the cases I want.